src/types/config.ts

```diff
@@ -92,7 +92,8 @@ export interface GlobalConfig extends EnvConfig {
 	WEB3_POLLING_INTERVAL: number;
 	SUBGRAPH_POLLING_INTERVAL: number;
 	TOKEN_PRECISION: number;
-
+	PRIMARY_NETWORK: any;
+	SECONDARY_NETWORK: any;
```

Comment:
we have mainnet and xdai concepts in GIVeconomy and I think it's better to use them.

Reply:
I think I mentioned something about this in a previous comment, can't seem to understand how to use it - in IO we do require the concepts of Primary vs Secondary networks

Reply:
We can have this terminology besides what we have in giveconomy. It can be Mainnet and xDAI config can be assigned to them and these two should have `BasicNetworkConfig` type

Reply:
We can fix this in the future.